src/components/SpinnerButton/SpinnerButton.tsx

```diff
@@ -74,6 +76,11 @@ const SpinnerButton: React.FC<SpinnerButtonProps> = ({
     animateWidth,
     animateHeight,
     animateRadius,
+    animationType
+  });
+  const { handleRipplePress, handleRippleLayout, animatedStyle } = useRippleButton({
```

Comment:
@rajanpayal-simform We should rename this useRippleButton to useSpinnerButton and also move that under SpinnerButton directory instead of utils